pages/_document.js

```diff
@@ -1,10 +1,7 @@
 import React from 'react'
 import Document, { Html, Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components'
-
-const SITE_DESCRIPTION = 'Bonding Curve Converter'
-const SITE_TITLE = 'Bonding Curve Converter'
-const SITE_URL = 'https://court.aragon.org/'
+import env from 'lib/environment'
 
 const ANALYTICS_CODE = `
   var _paq = window._paq || [];
```

Comment:
Hmm, we might remove these analytics from the initial release—will get back on this tomorrow.